lib/swaggerValidator/ajvDiscriminatorMap.ts

```diff
@@ -52,6 +52,8 @@ export const ajvEnableDiscriminatorMap = (ajv: Ajv, loader: JsonLoader) => {
               schemaPath: "/discriminator",
               schema: schemas,
               parentSchema,
+              _realSchema: schemas,
```

Comment:
Please add some comment about the issue to fix and the how is it comes from and how is used.